git.go

```diff
@@ -42,7 +41,7 @@ func (r *Repo) handshake() error {
 	if err != nil {
 		return fmt.Errorf("handshake: %v", err)
 	}
```

Comment:
Closing the response body (`defer resp.Body.Close()`) is missed and could be added here. Compare with https://github.com/golang/website/commit/f65c3a961a9862aa95b34c56633e20a0818a8218#diff-9d934ba1061efda849ec619df3059fefd1c813237d6a4a257b204adffea22078R42.